packages/graphiql-plugin-code-exporter/src/index.css

```diff
@@ -0,0 +1,129 @@
+.graphiql-code-exporter {
+  min-width: unset !important;
+  position: relative;
+  padding: var(--px-16) 0;
+}
+
+.graphiql-code-exporter > div {
+  font-family: var(--font-family) !important;
+  padding: 0 !important;
+  font-size: var(--font-size-body) !important;
+}
```

Comment:
We're using Vite for bundling this up, so we could use nesting to make the stylesheet a bit more concise and easy to read:
```css
.graphiql-code-exporter {
  min-width: unset !important;
  /* ... */

  & > div {
    font-family: var(--font-family) !important;
    /* ... */
  }

  /* ... */
}
```

Reply:
Are you sure? When converting the file it complains everywhere:

```shell
graphiql/packages/graphiql-plugin-code-exporter on  code-exporter is 📦 v0.0.1 via ⬢ v16.17.0 took 3s
➜ yarn build
yarn run v1.22.15
$ tsc --emitDeclarationOnly && node resources/copy-types.mjs && vite build
vite v2.9.13 building for production...
transforming (24) ../../node_modules/toggle-selection/index.jswarnings when minifying css:
▲ [WARNING] CSS nesting syntax is not supported in the configured target environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")

    <stdin>:5:2:
      5 │   & > div {
        ╵   ^
```

Reply:
I had to copy what `graphiql-react` was doing, using `postcss-nesting`